tests/vmi_configuration_test.go

```diff
@@ -1235,7 +1269,10 @@ var _ = Describe("[sig-compute]Configurations", func() {
 
 			It("[test_id:1676]should have attached a guest agent channel by default", func() {
 
-				agentVMI = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+				agentVMI = libvmi.NewAlpine(
+					libvmi.WithNetwork(v1.DefaultPodNetwork()),
+					libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
```

Comment:
Is it explicitly needed?

Reply:
Removed masquerade interfaces where not needed: 70d5a90